fluentd-elasticsearch env config doesn't render into daemonset

The `env` config variable in the fluentd-elasticsearch chart appears to not properly render into the daemonset.

Input:
values.yaml (as in the example Values.yaml in the chart):
```
env:
  WORKSPACE: "foo"
```

Steps:
```sh
helm fetch kiwigrid/fluentd-elasticsearch
helm template foo fluentd-elasticsearch -f values.yaml > foo.yaml
```

This results in the error: `coalesce.go:165: warning: skipped value for env: Not a table.`

I've also tried with a number of other formats for the Values.yaml file, just in case the example in the chart is wrong. But none of them resulted in a valid daemonset.

```
env:
  - WORKSPACE: "foo"
```
```
env:
  - key: "WORKSPACE"
    value: "foo"
```
```
env:
  - name: "WORKSPACE"
    value: "foo"
```

Using a list in any of these ways results in an environment variable with `name: 0` being added to the daemonset.

--- a/charts/fluentd-elasticsearch/Chart.yaml
+++ b/charts/fluentd-elasticsearch/Chart.yaml
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 6.2.2
+version: 6.2.3
 appVersion: 3.0.0
 home: https://www.fluentd.org/
 description: A Fluentd Helm chart for Kubernetes with Elasticsearch output

--- a/charts/fluentd-elasticsearch/values.yaml
+++ b/charts/fluentd-elasticsearch/values.yaml
@@ -80,19 +80,19 @@ fluentdArgs: "--no-supervisor -q"
 # If you want to add custom environment variables, use the env dict
 # You can then reference these in your config file e.g.:
 #     user "#{ENV['OUTPUT_USER']}"
-env:
+env: {}
   # OUTPUT_USER: my_user
   # LIVENESS_THRESHOLD_SECONDS: 300
   # STUCK_THRESHOLD_SECONDS: 900
 
 # If you want to add custom environment variables from secrets, use the secret list
-secret:
+secret: []
 # - name: ELASTICSEARCH_PASSWORD
 #   secret_name: elasticsearch
 #   secret_key: password
 
 # If you want to add plugins at start time
-additionalPlugins:
+additionalPlugins: []
 # - name: fluent-plugin-s3
 #   version: 1.1.11
 

